--- vivaldi-1.0.344.37/vivaldi.desktop~	2015-12-16 14:48:51.000000000 +0200
+++ vivaldi-1.0.344.37/vivaldi.desktop	2016-01-19 10:05:22.971946999 +0200
@@ -105,9 +105,9 @@
 Comment[zh_CN]=访问互联网
 Comment[zh_HK]=連線到網際網路
 Comment[zh_TW]=連線到網際網路
-Exec=/usr/bin/vivaldi-beta %U
+Exec=/usr/bin/vivaldi %U
 Terminal=false
-Icon=vivaldi-beta
+Icon=vivaldi
 Type=Application
 Categories=Network;WebBrowser;
-MimeType=text/html;text/xml;application/xhtml_xml;image/webp;x-scheme-handler/http;x-scheme-handler/https;x-scheme-handler/ftp;
+MimeType=text/html;text/xml;application/xhtml+xml;image/webp;x-scheme-handler/http;x-scheme-handler/https;x-scheme-handler/ftp;
